emission/core/wrapper/common_trip.py

```diff
@@ -0,0 +1,14 @@
+import logging
+import emission.core.wrapper.wrapperbase as ecwb
+
+class CommonTrip(ecwb.WrapperBase):
+    props = {"start_loc" : ecwb.WrapperBase.Access.WORM, # _id of common_place of starting point
+             "end_loc" : ecwb.WrapperBase.Access.WORM, # _id of common_place of ending point
+             "trips" : ecwb.WrapperBase.Access.WORM, # List of trip_ids that are associated with this common trip
```

Comment:
We didn't have it originally because there is no direct analog in mongo

On Thursday, February 4, 2016, shankari <notifications@github.com> wrote:

> In emission/core/wrapper/common_trip.py
> <https://github.com/e-mission/e-mission-server/pull/227#discussion_r51844538>
> :
>
> > @@ -0,0 +1,14 @@
> > +import logging
> > +import emission.core.wrapper.wrapperbase as ecwb
> > +
> > +class CommonTrip(ecwb.WrapperBase):
> > +    props = {"start_loc" : ecwb.WrapperBase.Access.WORM, # _id of common_place of starting point
> > +             "end_loc" : ecwb.WrapperBase.Access.WORM, # _id of common_place of ending point
> > +             "trips" : ecwb.WrapperBase.Access.WORM, # List of trip_ids that are associated with this common trip
>
> Why isn't there a similar list of places associated with the CommonPlace?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/e-mission/e-mission-server/pull/227/files#r51844538>.
>